### PR TITLE
KEYCLOAK-9191 Update documentation for client registration at Google

### PR DESCRIPTION
**Pull request by hmlnarik**

### CODE REVIEW

File: server_admin/topics/identity-broker/social/google.adoc [hunk 1]
```diff
@@ -24,50 +24,36 @@ Log in to the link:https://console.cloud.google.com/project[Google Developer Con
...
+`Authorized redirect URIs` field.  After you do this, click the `Create` button.
 
-When users log into Google from {project_name} they will see a consent screen from Google which will ask the user
-if {project_name} is allowed to view information about their user profile.  The next Google config screen asks
-you for information about this screen.
+.Create OAuth client ID
+image:images/google-create-oauth-id.png[]
 
-Once you click `Done` you will be brought to the `Credentials` page.  Click on your new OAuth 2.0 Client ID to view
+Once you click `Done` you will be brought to the `Credentials` page and the client ID.  Click on your new OAuth 2.0 Client ID to view
```
Comment (vmuzikar) [inline server_admin/topics/identity-broker/social/google.adoc]: > will be brought to the `Credentials` page and the client ID.

The sentence doesn't make much sense to me. The page seems to be called just `Credentials`, so I'd leave it as it was.

Also:
> Once you click `Done`

There's no `Done` button anymore.

**Review by vmuzikar — COMMENTED** [review]
Just a few minor things.

File: server_admin/topics/identity-broker/social/google.adoc (hunk 1 above)
Comment (hmlnarik) [reply server_admin/topics/identity-broker/social/google.adoc]: You're right, thanks for these!

**Review by vmuzikar — APPROVED** [review]
Thanks! LGTM now.

**Review by matthewhelmke — APPROVED** [review]
Thank you. LGTM

**Review by pdrozd — APPROVED** [review]

**Pull request merged**